pkg/cmd/join/scenario/join/default/klusterlets.cr.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright Contributors to the Open Cluster Management project
+apiVersion: operator.open-cluster-management.io/v1
+kind: Klusterlet
+metadata:
+  name: klusterlet
```

Comment:
I do not think the name here should be fixed. multiple hosted klusterlet can run in the same clusters.